pkg/agent/proxy/proxier.go

```diff
@@ -136,65 +136,66 @@ func (p *proxier) isInitialized() bool {
 	return p.endpointsChanges.Synced() && p.serviceChanges.Synced()
 }
 
-// removeStaleServices removes all expired Services. Once a Service is deleted, all
-// its Endpoints will be expired, and the removeStaleEndpoints method takes
-// responsibility for cleaning up, thus we don't need to call removeEndpoint in this
-// function.
+// removeStaleServices removes all the configurations of expired Services and their associated Endpoints.
 func (p *proxier) removeStaleServices() {
 	for svcPortName, svcPort := range p.serviceInstalledMap {
 		if _, ok := p.serviceMap[svcPortName]; ok {
 			continue
 		}
 		svcInfo := svcPort.(*types.ServiceInfo)
-		klog.V(2).Infof("Removing stale Service: %s %s", svcPortName.Name, svcInfo.String())
+		svcInfoStr := svcInfo.String()
+		klog.V(2).InfoS("Removing stale Service", "ServiceName", svcPortName, "ServiceInfo", svcInfoStr)
 		if err := p.ofClient.UninstallServiceFlows(svcInfo.ClusterIP(), uint16(svcInfo.Port()), svcInfo.OFProtocol); err != nil {
-			klog.ErrorS(err, "Failed to remove flows of Service", "Service", svcPortName)
+			klog.ErrorS(err, "Error when uninstalling ClusterIP flows for Service", "ServiceName", svcPortName, "ServiceInfo", svcInfoStr)
```

Comment:
The key "ServiceName" in such logs should be "servicePortName"?
BTW, the keys in the logs should not start with a capital according to the logs in other package?

Reply:
> The key "ServiceName" in such logs should be "servicePortName"?

I'll change that.

> BTW, the keys in the logs should not start with a capital according to the logs in other package?

I thought that `Service` in `ServiceName` means Kubernetes Service, and I capitalized them all.

Reply:
> @wenyingd do you have other comments?

No other comments.